src/client/components/ShoppingItem/ShoppingItem.js

```diff
@@ -47,7 +52,7 @@ export default function ShoppingItem({
         <Numberinput
           maxAvailable={quantity}
           getQuantity={setItemValue}
-          initValue={initValue}
+          initValue={itemValue}
```

Comment:
varsha , if this` initValue={initValue}` removes, so it is not possible to send this props to ShoppingItem and after NumberInput?

Reply:
itemValue is same as initValue. if you look at line no.17 initValue is used to initialize the "itemValue". so i dont understand how this can have any effect. I hope you have my latest code. 
 
the reason I changed it from initValue to itemValue, because with earlier changes, it was not showing the correct subtotal when rendering first time.

Reply:
I can change back.. but as i explained above it shall not have any effect

Reply:
push the changes as you suggested.. i will merge now

Reply:
ok Good Varsha. if it doesnot have effect so no problem. Good job perfect.